sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioUtils.scala

```diff
@@ -16,13 +16,36 @@
 
 package com.nvidia.spark.rapids
 
+import java.io.FileNotFoundException
+
 import org.apache.hadoop.fs.Path
 
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.catalyst.expressions.{Expression, PlanExpression}
 import org.apache.spark.sql.execution.datasources.{FileIndex, HadoopFsRelation, InMemoryFileIndex}
 import org.apache.spark.sql.execution.datasources.rapids.GpuPartitioningUtils
 
-object AlluxioUtils {
+
+object AlluxioUtils extends Logging {
+  val checkedAlluxioPath = scala.collection.mutable.HashSet[String]()
```

Comment:
If this is used only internally, better to be private. Since it is mutable.
```suggestion
  private val checkedAlluxioPath = scala.collection.mutable.HashSet[String]()
```